### PR TITLE
[Logs] Packetbeat

### PR DESCRIPTION
**Pull request by gavinelder**
to:
cc: @airbnb/streamalert-maintainers
related to:
resolves:

## Background

Add packetbeat schema 

Sorry no unit test's

### CODE REVIEW

File: conf/schemas/packetbeat.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "client_ip": "string",
+      "type": "string",
+      "transport": "string",
+      "bytes_in": "integer",
+      "bytes_out": "integer",
+      "dns": {}
+    },
+    "parser": "json",
+    "configuration": {
+      "optional_top_level_keys": ["bytes_in", "bytes_out"]
```
Comment (chunyong-lin) [inline conf/schemas/packetbeat.json]: Can you pretty print this line?
```
"optional_top_level_keys": [
  "bytes_in", 
  "bytes_out"
]
```

File: conf/schemas/packetbeat.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      "@timestamp": "string",
+      "start_time": "string",
+      "last_time": "string",
+      "type": "string",
+      "final": "boolean",
+      "dest": {},
+      "source": {}
+    },
+    "parser": "json",
+    "configuration": {}
```
Comment (chunyong-lin) [inline conf/schemas/packetbeat.json]: We can omit `configuration` field here.

**Comment by jack1902** [comment]
@chunyong-lin Should a requirement be added to the `contributing.rst` that when a new `schema` is added that a basic rule be setup to allow for passing in some test events ?

**Comment by gavinelder** [comment]
I need to make up some logs to trigger the rules.

**Comment by gavinelder** [comment]
@chunyong-lin hope this is of better quality :)

File: rules/community/packetbeat/packetbeat_blacklist_domain.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+DNS_BLACKLIST = [
+    'evil.com.'
+]
+
+@rule(logs=['packetbeat:dns'])
+def packetbeat_blacklisted_domain(rec):
+    """
+    author:       gavin (gavinelder)
+    description:  Lookup for BlackListed DNS (CnC).
```
Comment (chunyong-lin) [inline rules/community/packetbeat/packetbeat_blacklist_domain.py]: Can you add one more field to include the packetbeat documentation for users who haven't used packetbeat before.
```
reference: https://www.elastic.co/guide/en/beats/packetbeat/master/packetbeat-overview.html
```

Comment (gavinelder) [reply rules/community/packetbeat/packetbeat_blacklist_domain.py]: @chunyong-lin done.

**Pull request merged**